samples/word/02-paragraphs/get-paragraph-on-insertion-point.yaml

```diff
@@ -7,63 +7,71 @@ api_set:
     WordApi: 1.1
 script:
     content: |-
-        $("#get-paragraph").click(getParagraph);
-        $("#get-sentences").click(getSentences);
-        $("#setup").click(setup);
+        $("#get-paragraph").click(() => tryCatch(getParagraph));
+        $("#get-sentences").click(() => tryCatch(getSentences));
+        $("#setup").click(() => tryCatch(setup));
 
 
-        function getParagraph() {
-            Word.run(function (context) {
+        async function getParagraph() {
+            await Word.run(async (context) => {
                 // the collection of paragraphs of the current selection returns the full paragraphs contanied on it. 
-                var paragraph = context.document.getSelection().paragraphs.getFirst();
+                let paragraph = context.document.getSelection().paragraphs.getFirst();
+                paragraph.load("text");
                 context.load(paragraph);
```

Comment:
Is this context.load(paragraph); really needed, given that you have paragraph.load("text");  just above it?

Reply:
good catch! local fixed